src/NuGet.Services.EndToEnd/Support/TestSettings.cs

```diff
@@ -56,6 +58,12 @@ private TestSettings()
         /// </summary>
         public bool SkipGalleryTests { get; set; }
 
+        public bool IsTestingAzureSearchService()
```

Comment:
Can you create an issue to clean this fact up later when we are off old search?

Reply:
https://github.com/NuGet/Engineering/issues/2636